scripts/fetch-configlet_v3

```diff
@@ -1,8 +1,8 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Here I've squeezed in an unrelated change. See https://github.com/exercism/configlet/commit/e13f7c74c1d6840ecf92e746aa32dec12cb5febb